pkg/volume/emptydir/empty_dir.go

```diff
@@ -397,9 +411,14 @@ func (ed *emptyDir) TearDownAt(dir string) error {
 }
 
 func (ed *emptyDir) teardownDefault(dir string) error {
+	// Remove any quota
+	err := quota.ClearQuota(ed.mounter, dir)
```

Comment:
is anything lingering if this fails and we remove the directory?

Reply:
The filesystem will still know about the quota ID and associated quotas, but it won't be tied to anything.  There's nothing we can do in this circumstance, though.